### PR TITLE
Attempt to parse string initial ratings.

### PR DESCRIPTION
**Pull request by animate-object**
Sometimes data coming in from SurveyJS is a string like "5" or "6.0". Let's make the widget more resilient by trying to parse those values before falling back to `0.0`.

### CODE REVIEW

**Comment by animate-object** [comment]
I notice this and the last merge to master are failing due to a missing circle ci config. Assuming no action needed.

File: src/index.js [hunk 1]
```diff
@@ -35,13 +35,26 @@ export default class VASSlider extends React.PureComponent {
...
+      rating: this.parseInitialRating(initialRating)
     };
   }
 
+  parseInitialRating = (initialRating) => {
+    if (typeof initialRating === "number") {
+      return this.translateRating(initialRating)
+    } else if (typeof initialRating === "string") {
+      try {
+        parseFloat(initialRating)
```
Comment (slycoder) [inline src/index.js:49]: Don't you need to pass the result somewhere?

Comment (animate-object) [reply src/index.js]: Yup! Just caught this. Should've tested before submitting for review.

Comment (animate-object) [reply src/index.js]: Updated.

**Review by slycoder — APPROVED** [review]

**Pull request merged**